packages/test-commerce-sdk-react/app/pages/use-shopper-product.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+// @ts-ignore
+import {useProduct} from 'commerce-sdk-react'
+import Json from '../components/Json'
+import {Link, useParams} from 'react-router-dom'
+
+function UseShopperProduct() {
+    const {productId}: {productId: string} = useParams()
+    const {data, isLoading, error} = useProduct({
+        parameters: {
+            id: productId
+        }
+    })
```

Comment:
For others to use our hooks, it looks like they need to type `{parameters: {...}}` for the argument each time. The other option that the isomorphic sdk allows is `{header}`. But would users ever need to pass in a header option?

If they would use only `parameters`, what if we abstract that out and make it easier for them to pass in the product id? So it would simply be `useProduct({id})`.

/cc @kevinxh @wjhsf

Reply:
Good point, 🤔 . IMO, we can break it down into two arguments. For example
```
const useProduct(paramters, deps, opts) whether opts allows `headers` objects, and it is optional
```

Reply:
@vmarta I've separated the interface as above, see it [here](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/681/files#diff-77a7b2187310a9f065844782966ae020d8c5e42c99e5171a5928963869223417L7)